engine_bs/README.md

```diff
@@ -15,17 +15,53 @@ sh <(curl -sL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.
 opam init
 ```
 
-2. Cloning repositories
+2. BuckleScript
+
+Clone the BuckleScript fork that works with Sketch.sh. `bucklescript` root project folder
+should be next to `sketch-sh` root project folder in the directory structure.
 
 ```sh
-git clone https://github.com/BuckleScript/bucklescript
-git clone https://github.com/BuckleScript/bucklescript-playground
-git clone https://github.com/facebook/reason/
+git clone https://github.com/thangngoc89/bucklescript/
+git checkout sketch
+```
+
+js_of_ocaml is also needed to build the browser version of the BuckleScript compiler:
+
+```sh
+opam install js_of_ocaml
+```
+
+_Note: js_of_ocaml can be installed in any opam switch, even different than 4.02.3._
```

Comment:
This is incorrect. To build bs's repl, you need `4.02.3` switch because `cmi` and `cmt` format are different in each compiler versions

Reply:
Are you sure? I have successfully compiled the repl with ocaml `4.05.0`, I think this version only affects the jsoo binary. The `cmi_files` are already available, and jsoo just produce the bytecode for them.

I can change this though if it's really needed, I just thought it'd be good to mention this as it worked for me and it avoids more "opam switch" confusion.

Reply:
Hmm, `js_of_ocaml` is the one responsible for turning the cmi files to hex sequenses in Js. So maybe it doesn't ready the cmi files at all, it just turn binary -> hex. But as a rule of thumb when working with bucklescript, use `4.02.3` things may or may not work if you're using another switch